crowbar_framework/app/assets/stylesheets/bootstrap.scss

```diff
@@ -1,7 +1,19 @@
+/*!
```

Comment:
Move all this content into `bootstrap-sprockets` and leave this file for bootstrap extensions.